src/scripts/dom.ts

```diff
@@ -0,0 +1,244 @@
+import { IForecastRoot, ISeries } from "../models/forecast";
+
+export function setDOM(results: IForecastRoot, city: string): void {
+  if (!(results?.dataseries && results?.dataseries[0])) {
+    throw new Error("No weather results found.");
+  }
+  setTodayElement(results.dataseries[0], city);
+  set7dayForecast(results.dataseries);
+}
+
+function setTodayElement(todaysWeatherData: ISeries, city: string): void {
+  if (
+    !(
+      todaysWeatherData?.date &&
+      todaysWeatherData?.temp2m?.min &&
+      todaysWeatherData?.temp2m?.max &&
+      todaysWeatherData?.weather
+    )
+  ) {
+    throw new Error("Today's weather data is not found.");
+  }
+
+  // todays date
+  const todaysDate: Date = getDate(todaysWeatherData.date.toString());
+  const monthNames: string[] = [
+    "January",
```

Comment:
I mentioned to Jules but you can move the `monthNames` and `days` to their own file. They are static and can just be imported where used.